src/main/conversions/v2.21.0/c2_21_0_2018050101.clj

```diff
@@ -0,0 +1,44 @@
+(ns facepalm.c2-21-0-2018050101
+  (:use [kameleon.sql-reader :only [load-sql-file exec-sql-statement]]))
+
+(def ^:private version
+  "The destination database version."
+  "2.21.0:20180501.01")
+
+(defn- add-uniqueness-constraint
+  []
+  (exec-sql-statement
+   "ALTER TABLE ONLY interactive_apps_proxy_settings
+    ADD CONSTRAINT interactive_apps_proxy_settings_id_unique UNIQUE (id)"))
```

Comment:
I thought the `UNIQUE` constraint was automatically added to primary keys?

Reply:
When I tested creating the database from scratch it was barking at me about the missing uniqueness constraint.

Reply:
Ah, that's probably because `00_71_container_settings_fkey.sql` is loaded first, before loading `84_interapps_proxy_settings.sql` which defines the primary key (which would make it unique). If the constraint in `00_71_container_settings_fkey.sql` is moved to `71_container_settings.sql`, and `84_interapps_proxy_settings.sql` is renamed to `00_84_interapps_proxy_settings_pkey.sql`, then these files will be loaded in the correct order.